src/Unity/ubuntulocationservice.cpp

```diff
@@ -64,7 +64,9 @@ UbuntuLocationService::UbuntuLocationService(const GeoIp::Ptr& geoIp)
     : m_geoIp(geoIp)
 {
     // If the location service is disabled
-    if (qEnvironmentVariableIsSet("UNITY_SCOPES_NO_LOCATION"))
+    // if (qEnvironmentVariableIsSet("UNITY_SCOPES_NO_LOCATION"))
+    // HACK: do not ask for location, as nobody really uses scopes any more.
+    if (true)
```

Comment:
Not to be "that guy", why didn't you just remove the if :)

Reply:
if you want to save that one cpu cycle, sure, we can do that. wanted to keep the structure intact in case anyone would reuse the code at some point, but it
s probably a bad idea to reuse it, anyways, so... Idk. We could have also set the env var, btw, but then we'd have to unset it later...